source/client-side-encryption/client-side-encryption.rst

```diff
@@ -563,6 +563,7 @@ accept arbitrary strings at runtime for forward-compatibility.
 
    interface KMIPKMSOptions {
       endpoint: string;
+      delegated: boolean; // If true (recommended), the KMS provider will perform DEK encryption and decryption. Defaults to false.
```

Comment:
Can either `KMIPKMSOptions.delegated` or `masterKey.delegated` options be removed? I expect both are not needed for this feature.

`KMIPKMSOptions.delegated` can be true, false, or unset. `masterKey.delegated` can be true, false, or unset. Including both options may require additional testing and documentation to describe which option applies when they have differing values:

  | KMIPKMSOptions.delegated true | KMIPKMSOptions.delegated false | KMIPKMSOptions.delegated unset
-- | -- | -- | --
**masterKey.delegated true** | Delegated. | ? | Delegated.
**masterKey.delegated false** | ? | Not delegated. | Not delegated.
**masterKey.delegated unset** | Delegated. | Not delegated. | Not delegated.

</div></b>